crates/sui-config/src/node.rs

```diff
@@ -616,7 +616,7 @@ impl AuthorityStorePruningConfig {
             max_transactions_in_batch: 1000,
             periodic_compaction_threshold_days: None,
             num_epochs_to_retain_for_checkpoints,
-            enable_pruning_tombstones: false,
+            enable_pruning_tombstones: true,
```

Comment:
This is only to facilitate the test in this PR. (don't feel too strong to add an option in test cluster to set this, since it'll be removed soon anyway) I'll make enable_pruning_tombstones set to true by default in the next PR.